### DISCUSSION TITLE
我是乌蒙吃

### DISCUSSION THREAD

**Discussion by AveryJiang** [share121/inter-knot#5226] [其他]
![b50](https://github.com/user-attachments/assets/268533e3-314b-4548-b939-316fd72728ac)
刚刚入坑 迷上了舞萌DX 打了50pc

**Comment by HLXS-HLXS** [comment]
如果这是在其他网站，网友会说你不行，但在绳网，我会说你非常牛逼<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by KnowPigeon** [comment]
哎wmc

**Comment by aino2333** [comment]
我这里机厅关门了<img src="https://img-static.mihoyo.com/communityweb/upload/7bd889870907d2d96673d09afb53cf63.png" alt="三月七_哭" width="75" >

**Comment by WHTANUKI** [comment]
唉舞萌吃。。。
但有一说一，比我强（膜拜ing
